website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
+| `notification_configuration_id`              | string | ID of the configuration which defined this notification.                  |
+| `notification_configuration_url`             | string | URL of the Notification Configuration.                                    |
```

Comment:
Does this take you to it in TFC? Where does the URL go?

Reply:
How about:

"URL to get the notification configuration from the Terraform Cloud API."